tests/observability/metrics/test_network_metrics.py

```diff
@@ -47,3 +56,44 @@ def test_kubevirt_vmi_network_traffic_bytes_total(
             vm=vm_for_test,
             interface_name=vm_for_test_interface_name,
         )
+
+    """Windows vms"""
```

Comment:
why not split into 2 test classes so it will be more visible what is tested (linux/windows)?